src/webapi/element.rs

```diff
@@ -225,6 +226,40 @@ pub trait IElement: INode + IParentNode + IChildNode {
     fn insert_html_after( &self, html: &str ) -> Result<(), InsertAdjacentError> {
         self.insert_adjacent_html(InsertPosition::AfterEnd, html)
     }
+
+    /// The slot property of the Element interface returns the name of the shadow DOM
+    /// slot the element is inserted in.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Element/slot)
+    // https://dom.spec.whatwg.org/#dom-element-slot
```

Comment:
Always link to the IDL when linking to the specs, in this case this should be:

`https://dom.spec.whatwg.org/#ref-for-dom-element-slot`

The same applies for other links. (: